internal/operands/common-templates/reconcile.go

```diff
@@ -0,0 +1,145 @@
+package common_templates
+
+import (
+	"fmt"
+	templatev1 "github.com/openshift/api/template/v1"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"kubevirt.io/ssp-operator/internal/common"
+	"kubevirt.io/ssp-operator/internal/operands"
+	"path/filepath"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sync"
+)
+
+var (
+	loadTemplatesOnce sync.Once
+	templatesBundle   []templatev1.Template
```

Comment:
I don't like accessing this variable from multiple files in the package, I would move it to `resource.go`, and make `readTemplates()` populate it if it's empty, or just return it if it is already initialized

Reply:
The `readTemplates()` funciton on line 121 could return the loaded template slice, instead of assigning it to the global variable. Then both these variables `loadTemplatesOnce` and `templatesBundle` would be accessed only from one file.

Reply:
Changed it this way